### PR TITLE
fix(vscode-deb): add arm architecture

### PR DESCRIPTION
**Pull request by Diegiwg**

### CODE REVIEW

File: packages/vscode-deb/vscode-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+    hash="4d7f0cc976ed1dfde1ceeaadd16829427a477eba1b49ff82dd490395f32eb78c"
+    ;;
+  *) return 1 ;;
+esac
+
+homepage="https://code.visualstudio.com/"
 description="Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux."
-hash="eeca823ab932d927a12dc85fb2f7fb011e25d8bf61bcb96e807c9add57e9ccd0"
-arch=('amd64')
+
```
Comment (Henryws) [inline packages/vscode-deb/vscode-deb.pacscript:24]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Diegiwg** [comment]
@Henryws, could you explain to me the problem with `pre-commit`? I'm not understanding the error.

File: packages/vscode-deb/vscode-deb.pacscript [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+    ;;
+  armhf)
+    url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}-1683730415_$CARCH.deb"
+    hash="4d7f0cc976ed1dfde1ceeaadd16829427a477eba1b49ff82dd490395f32eb78c"
+    ;;
+  *) return 1 ;;
+esac
+
+homepage="https://code.visualstudio.com/"
 description="Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux."
```
Comment (Henryws) [inline packages/vscode-deb/vscode-deb.pacscript:23]:
```suggestion
description="lightweight but powerful source code editor"
```

File: packages/vscode-deb/vscode-deb.pacscript [hunk 3]
```diff
@@ -1,8 +1,25 @@
 name="vscode-deb"
+arch=("amd64" "arm64" "armhf")
 gives="code"
-version="1.78.2-1683731010"
-url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}_amd64.deb"
+version="1.78.2"
+
```
Comment (Henryws) [inline packages/vscode-deb/vscode-deb.pacscript:5]:
```suggestion
```

File: packages/vscode-deb/vscode-deb.pacscript [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+    url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}-1683731061_$CARCH.deb"
+    hash="4d7f0cc976ed1dfde1ceeaadd16829427a477eba1b49ff82dd490395f32eb78c"
+    ;;
+  armhf)
+    url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}-1683730415_$CARCH.deb"
+    hash="4d7f0cc976ed1dfde1ceeaadd16829427a477eba1b49ff82dd490395f32eb78c"
+    ;;
+  *) return 1 ;;
+esac
+
```
Comment (Henryws) [inline packages/vscode-deb/vscode-deb.pacscript:21]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**